### DISCUSSION TITLE
Set TTL for vertex

### DISCUSSION THREAD

**Discussion by dinhphu2k1-gif** [JanusGraph/janusgraph#3529] [Q&A] [answered]
I have a janusgraph server and it ran about 4 months. Can i also set ttl for it?

**Comment by dinhphu2k1-gif** [comment, marked as answer]
mgmt.getVertexLabel('tweet').setTTL(...)

**Comment by vtslab** [comment]
You mean like this: https://docs.janusgraph.org/schema/advschema/#vertex-ttl
